websites/management/commands/markdown_cleaning/parsing_utils.py

```diff
@@ -72,73 +72,43 @@ def scan_string(self, string: str):
         return self.grammar.scanString(string)
 
 
-def standardize_title(s, l, toks):
-    text: str = toks[0]
-    if text.startswith("'") and text.endswith("'"):
-        double_quoted = (
-            text[1:-1]  # remove the outer single quote
-            .replace("\\'", "'")  # unescape single quotes
-            .replace('"', '\\"')  # escape double quotes
-        )
-        return json.loads(double_quoted)
-    elif text.startswith('"') and text.endswith('"'):
-        return json.loads(text[1:-1])
+def escape_double_quotes(s: str):
+    """Encase `s` in double quotes and escape double quotes within `s`."""
+    return s.replace('"', '\\"')
 
 
-def unescape_single_quoted_string(text: str):
+def unescape_string_quoted_with(text: str, single_quotes=False):
```

Comment:
@gumaerc This change (see https://github.com/mitodl/ocw-studio/pull/1165/commits/d062254cf2b77b3d38604bd4348e505819ab7f3b) is a new commit after the rebase.

Using json.loads/dumps for escaping/unescaping stuff was not a great idea. I found this was doing unwanted things, like converting `"«"` to `"\\u00ab"`. BTW: My goal with the unescaping was that
- `Shortcode(name='whatever', args=['This has "quotes" in it']).to_hugo()` should handle the escaping for you...
- So the internal shortcode representation should be unescaped, so the parser needs to unescape args when creating the shortcodes. 

The extra escaping from json dumps/loads should not have caused any problems on prod yet. The two places this was used were
- link titles `[text](destination "title")` which tend not to have quotes in the anyway / aren't really supported in ocw-next yet
- the shortcode parser has only been used so far for removing extra arguments from `< resource >`